UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/AccessibilityStatement/Index.cshtml

```diff
@@ -0,0 +1,50 @@
+    </ul>
+
+    <p>We’ve also made the website text as simple as possible to understand.</p>
+    <p><a href="https://abilitynet.org.uk/" target="_blank">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    <h2>Feedback and contact information</h2>
+    <p>If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:</p>
+    <ul class="msi-list-style">
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Email <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a></li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Call +44 (0)1823 484444</li>
+    </ul>
+
+    <p>We’ll consider your request and reply to you within 5 working days.</p>
+    <h3>Reporting accessibility problems with this website</h3>
+    <p>We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a>to let us know your concerns.</p>
+    <h3>Enforcement procedure</h3>
+    <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the Equality Advisory and Support Service (EASS).
+    </p>
+    <h3>Technical information about this website’s accessibility</h3>
+    <p>The UK Hydrographic Office is committed to making this website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+    <h3>Compliance status</h3>
+    <p>This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.</p>
+    <h3>Non-accessible content</h3>
+    <p>The content listed below is non-accessible for the following reasons.</p>
+    <h3>Non-compliance with the accessibility regulations</h3>
+    <p>N/A</p>
+
+    <h3>How we tested this service </h3>
+    <p>The service was last tested on 21 july 2022 and was checked for compliance with WCAG 2.1 A. The test was carried out by us and our supplier.</p>
+    <p>User journeys through the website were tested with a range of common assistive technologies.</p>
+    <h3>What we’re doing to improve accessibility</h3>
+    <p><a href="https://github.com/UKHO/docs/blob/main/quality-assurance/web-accessibility-testing.md" target="_blank">Accessibility Testing Approach at UKHO</a> documents show how we plan to test and improve accessibility on this website and all our products and services. We reviewed these documents on 23 May 2021 and the next review will be on 1 November 2021.</p>
```

Comment:
Please update the last statement as per suggestion given by Brent.
```suggestion
    <p><a href="https://github.com/UKHO/docs/blob/main/quality-assurance/web-accessibility-testing.md" target="_blank">Accessibility Testing Approach at UKHO</a> documents show how we plan to test and improve accessibility on this website and all our products and services. We review these documents on a regular basis.</p>
```

Reply:
done required changes